### PR TITLE
Stretch fixes

### PR DESCRIPTION
**Pull request by dwalton76**

### CODE REVIEW

**Comment by dwalton76** [comment]
Resolves #339

File: tests/README.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
+If you do not have the `fake-sys` directory it means you did not use the
+`--recursive` option when you git cloned the ev3dev-lang-python repo.
+Example:
+
+```
+$ git clone --recursive https://github.com/rhempel/ev3dev-lang-python.git
```
Comment (WasabiFan) [inline tests/README.md]: You can run `git submodule init` to do it post-clone.

File: tests/README.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
 Commands used to copy the /sys/class node:
 
-```sh
-node=lego-sensor/sensor0
-mkdir -p ./${node}
-# Copy contents of special files, do not follow symlinks:
-cp -P --copy-contents -r /sys/class/${node}/* ./${node}/
+```
+$ node=lego-sensor/sensor0
+$ mkdir -p ./${node}
```
Comment (WasabiFan) [inline tests/README.md:25]: I think this should be at the bottom, because it's rarely needed. The important command is the one  for running the tests.

**Review by WasabiFan — APPROVED** [review]

File: ev3dev/core.py [hunk 3]
```diff
@@ -42,6 +42,7 @@
 import fnmatch
 import numbers
 import array
+import logging
```
Comment (ddemidov) [inline ev3dev/core.py:45]: I wonder how much does this add to startup time.

Comment (dwalton76) [reply ev3dev/core.py:45]: No noticable difference as far as I can tell

File: tests/README.md [hunk 4]
```diff
@@ -1,20 +1,27 @@
-If you do not have the `fake-sys` directory it means you did not use the
-`--recursive` option when you git cloned the ev3dev-lang-python repo.
-Example:
+# fake-sys directory
+The tests require the fake-sys directory which comes from
+https://github.com/ddemidov/ev3dev-lang-fake-sys
+
+If you have already cloned the ev3dev-lang-python repo but do not have the
+`fake-sys` directory use `git submodule init` to get it.  If you have not
+already closed the ev3dev-lang-python repo you can use the `--recursive` option
```
Comment (WasabiFan) [inline tests/README.md]: `closed` -> `cloned`

**Pull request merged**